### ISSUE TITLE
1.1.24: Flaky unit test dsp.3d.plane

### ISSUE THREAD

**Issue by dvzrv** [sadko4u/lsp-plugins#129]
Hi! When packaging 1.1.24 for Arch Linux I had to rebuild a few times to test things with #128 
On a recent build I ran the tests with

```
make test
./.test/lsp-plugins-test utest --nofork --debug --verbose
```

and got:

```
[..]

Testing native::base64_enc...
Testing native::base64_dec...
decoded = Man is distinguished, not only by his reason, but by this singular passion from other animals, which is a lust of the mind, that by a perseverance of delight in the continued and indefatigable generation of knowledge, exceeds the short vehemence of any carnal pleasur
Testing native::base64_encdec...
Testing encoding of string ""...
Testing decoding of string ""...
Testing encoding of string "1"...
Testing decoding of string "MQ"...
Testing encoding of string "12"...
Testing decoding of string "MTI"...
Testing encoding of string "123"...
Testing decoding of string "MTIz"...
Testing encoding of string "Test string for base64 encoding"...
Testing decoding of string "VGVzdCBzdHJpbmcgZm9yIGJhc2U2NCBlbmNvZGluZw"...
Testing encoding of string "0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ+/"...
Testing decoding of string "MDEyMzQ1Njc4OWFiY2RlZmdoaWprbG1ub3BxcnN0dXZ3eHl6QUJDREVGR0hJSktMTU5PUFFSU1RVVldYWVorLw"...
Testing reverse_bits u8...
Testing reverse_bits i8...
Testing reverse_bits u16...
Testing reverse_bits i16...
Testing reverse_bits u32...
Testing reverse_bits i32...
Testing reverse_bits u64...
Testing reverse_bits i64...
Testing int_log2 u8...
Testing int_log2 i8...
Testing int_log2 u16...
Testing int_log2 i16...
Testing int_log2 u32...
Testing int_log2 i32...
Testing int_log2 u64...
Testing int_log2 i64...
Testing atomic_cas...
Testing atomic_swap...
Testing atomic_add...
Testing sse_vector
Launching native_ck_triplet implementation
Checking triangle 0
Checking triangle 1
Checking triangle 2
Launching sse_ck_triplet implementation
Checking triangle 0
Checking triangle 1
Checking triangle 2
Testing native::ck_triangle...
ck=-26.000000
ck=-3072.000000
Testing sse::ck_triangle...
ck=-26.000000
ck=-3072.000000
Testing native::split_triangle_raw...
Testing sse::split_triangle_raw...
Testing sse3::split_triangle_raw...
Testing sse::calc_split_point_p2v1...
Testing sse::calc_split_point_pvv1...
Testing sse3::calc_split_point_p2v1...
Testing sse3::calc_split_point_pvv1...
Testing sse_point
Testing sse::calc_plane_p3...
Testing sse::calc_plane_pv...
pv[0]: { -7.614584e+00, 7.205029e+00, -4.932916e+00, 1.000000e+00 }
pv[1]: { 9.377472e+00, 3.864678e+00, 9.394796e+00, 1.000000e+00 }
pv[2]: { 9.150135e+00, 3.829090e+00, 9.204185e+00, 1.000000e+00 }
v[0]: { 6.434078e-01, -1.029514e-02, -7.654544e-01, 1.197537e+00 }
v[1]: { 6.434074e-01, -1.029192e-02, -7.654548e-01, 1.197509e+00 }
w[0] = 1.782082e+00, w[1] = 1.782085e+00
Unit test 'dsp.3d.plane' has failed at file utest/dsp/3d/plane.cpp, line 105 with message:
  result of functions differ
```

I'm building this on one of our build servers:

```
$ lscpu
Architecture:                    x86_64
CPU op-mode(s):                  32-bit, 64-bit
Byte Order:                      Little Endian
Address sizes:                   43 bits physical, 48 bits virtual
CPU(s):                          48
On-line CPU(s) list:             0-47
Thread(s) per core:              2
Core(s) per socket:              24
Socket(s):                       1
NUMA node(s):                    4
Vendor ID:                       AuthenticAMD
CPU family:                      23
Model:                           1
Model name:                      AMD EPYC 7401P 24-Core Processor
Stepping:                        2
Frequency boost:                 enabled
CPU MHz:                         1328.548
CPU max MHz:                     2000.0000
CPU min MHz:                     1200.0000
BogoMIPS:                        4001.08
Virtualization:                  AMD-V
L1d cache:                       768 KiB
L1i cache:                       1.5 MiB
L2 cache:                        12 MiB
L3 cache:                        64 MiB
NUMA node0 CPU(s):               0-5,24-29
NUMA node1 CPU(s):               6-11,30-35
NUMA node2 CPU(s):               12-17,36-41
NUMA node3 CPU(s):               18-23,42-47
Vulnerability Itlb multihit:     Not affected
Vulnerability L1tf:              Not affected
Vulnerability Mds:               Not affected
Vulnerability Meltdown:          Not affected
Vulnerability Spec store bypass: Mitigation; Speculative Store Bypass disabled via prctl and seccomp
Vulnerability Spectre v1:        Mitigation; usercopy/swapgs barriers and __user pointer sanitization
Vulnerability Spectre v2:        Mitigation; Full AMD retpoline, IBPB conditional, STIBP disabled, RSB filling
Vulnerability Srbds:             Not affected
Vulnerability Tsx async abort:   Not affected
Flags:                           fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush mmx fxsr sse sse2 ht syscall nx mmxext fxsr_opt pdpe1gb rdtscp lm constant_tsc rep_good nopl nonstop_tsc cpuid extd_apicid amd_dcm aperfmperf pni pclmulqdq monitor ssse3 fma cx16 sse4_1 sse4_2 movbe popcnt
                                 aes xsave avx f16c rdrand lahf_lm cmp_legacy svm extapic cr8_legacy abm sse4a misalignsse 3dnowprefetch osvw skinit wdt tce topoext perfctr_core perfctr_nb bpext perfctr_llc mwaitx cpb hw_pstate sme ssbd sev ibpb vmmcall fsgsbase bmi1 avx2 smep bmi2 rdseed adx smap clflushopt sha_ni xs
                                 aveopt xsavec xgetbv1 xsaves clzero irperf xsaveerptr arat npt lbrv svm_lock nrip_save tsc_scale vmcb_clean flushbyasid decodeassists pausefilter pfthreshold avic v_vmsave_vmload vgif overflow_recov succor smca
```

**Comment by sadko4u** [comment]
```
v[0]: { 6.434078e-01, -1.029514e-02, -7.654544e-01, 1.197537e+00 }
v[1]: { 6.434074e-01, -1.029192e-02, -7.654548e-01, 1.197509e+00 }
```

There is some computation imprecision of the 4th vector component. Will lower the precision check.

**Comment by sadko4u** [comment]
@dvzrv please verify that it builds OK now on EPYC processors.

**Comment by sadko4u** [comment]
Merged changes into **devel**

**Comment by sadko4u** [comment]
Removed branch **github-issue-129**

**Comment by sadko4u** [comment]
Available in 1.1.26 release
